src/json/program/AfterUserCreatedOrUpdatedTriggerContext.schema.ts

```diff
@@ -0,0 +1,34 @@
+import { JSONSchema6 } from "json-schema";
+import {
+  userContext,
+  baseUserContextFields,
+  contextTimeField,
+  customFieldsAndEventsDefinitions
+} from "./ProgramTriggerVariableContext.schema";
+
+const afterUserCreatedOrUpdatedTriggerContextSchema: JSONSchema6 = {
+  $schema: "http://json-schema.org/draft-06/schema#",
+  type: "object",
+  title: "After User Created Or Updated Trigger Context",
+  description:
+    "Defines the trigger variables available to an AFTER_USER_CREATED_OR_UPDATED trigger",
+  properties: {
+    type: {
+      enum: ["AFTER_USER_CREATED_OR_UPDATED"]
+    },
+    user: {
+      title: "Current User Data",
+      ...userContext
+    },
+    previous: {
+      title: "Previous User Data",
```

Comment:
```suggestion
      title: "Previous User",
```

Reply:
@jorgecgll and @rebeccachan looking for your feedback on the labels for these tier variables.

These labels will show up in some way in the portal (for picking custom fields for example), in our docs (similar to our existing Theme Variables used in handlebars), in logs, error messages, etc.

Important to get them right